app/src/androidTest/java/com/sdp/movemeet/viewTest/chatTest/ChatActivityTest.java

```diff
@@ -57,7 +48,7 @@
 @RunWith(AndroidJUnit4.class)
 public class ChatActivityTest {
 
-    private FirebaseDatabase mDatabase;
+    private FirebaseDatabase firebaseDatabase;
```

Comment:
Nice rename

Reply:
Yes, according to our convention, we have to avoid using the prefix "m" in front of our variable names.